src/fusor/examples/create_examples.py

```diff
@@ -0,0 +1,240 @@
+"""Script for creating JSON examples
+
+To generate all examples: python3 src/fusor/examples/create_examples.py
```

Comment:
should we add something in the readme about this script? I'm fine whether we decide to add it or not

Reply:
I think a new subhead in contributing.rst would be appropriate. Just a line acknowledging this workflow's existence is enough imo

Reply:
@jsstevenson just saw this comment